configure.ac

```diff
@@ -628,16 +628,12 @@ dnl Checks for library functions.
 AC_TYPE_GETGROUPS
 AC_PROG_GCC_TRADITIONAL
 AC_FUNC_MEMCMP
-AC_FUNC_VPRINTF
-AC_CHECK_FUNCS(fseeko getdomainname gethostname gettimeofday lckpwdf mkdir select)
-AC_CHECK_FUNCS(strcspn strdup strspn strstr strtol uname)
+AC_CHECK_FUNCS(getdomainname lckpwdf)
 AC_CHECK_FUNCS(getutent_r getpwnam_r getpwuid_r getgrnam_r getgrgid_r getspnam_r getmntent_r)
 AC_CHECK_FUNCS(getgrouplist)
-AC_CHECK_FUNCS(inet_ntop inet_pton innetgr)
-AC_CHECK_FUNCS(quotactl)
-AC_CHECK_FUNCS(unshare)
+AC_CHECK_FUNCS(innetgr)
 AC_CHECK_FUNCS(explicit_bzero memset_explicit)
-AC_CHECK_FUNCS([ruserok_af ruserok], [break])
```

Comment:
$ac_cv_func_ruserok is still used to define COND_BUILD_PAM_RHOSTS

Reply:
It's back in.